tests/e2e/basic/_basic_test_setup/setup_e2e_bridge.py

```diff
@@ -0,0 +1,181 @@
+import pytest
+
+
+@pytest.mark.sanity
+@pytest.mark.setup_bridge
+@pytest.mark.bridge
+@pytest.mark.parametrize(
```

Comment:
Is there documentation of parametrize behaviour? Maybe a readme on how to add a new test case, new test class and new test module. Then adding a new marker etc.